source/fuzz/fuzzer_util.cpp

```diff
@@ -627,6 +627,7 @@ void AddGlobalVariable(opt::IRContext* context, uint32_t result_id,
       context, SpvOpVariable, type_id, result_id, std::move(operands)));
 
   AddVariableIdToEntryPointInterfaces(context, result_id);
+  UpdateModuleIdBound(context, result_id);
```

Comment:
This seems to be a nice improvement and I'm fine with it being part of this PR.  However, do feel free to do very small PRs that contribute these kinds of improvements in isolation - that's slightly preferable.

Reply:
Will do!